internal/app/jdk/detect.go

```diff
@@ -26,12 +27,17 @@ func findWithMinVersion(jdkPaths []string, minVersion int) (string, bool) {
 		return "", false
 	}
 
+	homeDir, err := os.UserHomeDir()
+	if err != nil {
+		return "", false
```

Comment:
We should think about log file and collecting logs

Reply:
They are generally collected but not here.